pages/_sites/[site]/tinycms/homepage.js

```diff
@@ -72,13 +72,8 @@ export default function HomePageEditor({
     let article2 = null;
     let article3 = null;
 
-    if (selectedLayout.name !== 'Big Featured Story' && subFeaturedTopArticle) {
+    if (selectedLayout.name === 'Large Package Story Lead') {
```

Comment:
I think we still want to include the `&& subFeaturedTopArticle` and `subFeaturedBottomArticle` check in the if-statement. Because if we try to access the id's here when either is null/undefined, we will throw an error.